### ISSUE TITLE
Landing page

### ISSUE THREAD

**Issue by Git-Jake-Hill** [Git-Jake-Hill/potionfy#1]
Write a simple landing page message on the index.html file.

**Comment by Duymai110** [comment]
i just copy the code directly from the website straight to the  index.html file

**Comment by Duymai110** [comment]
how do i push it or u have to push it?

**Comment by Git-Jake-Hill** [comment]
I'm not sure what you are up too, but I think you need to do a pull request.

**Comment by Duymai110** [comment]
i'm so lost.bruh 
sorry

**Comment by Git-Jake-Hill** [comment]
Ah no worries, it was a bit hard to follow under pressure.
I think you did make the change, but you just needed to create a branch first, then change the file on the branch and merge the branch. all good though.

**Comment by Git-Jake-Hill** [comment]
close
